Formula/libomxil-bellagio.rb

```diff
@@ -12,27 +12,35 @@ class LibomxilBellagio < Formula
 
   option "without-test", "Skip compile-time tests"
   option "with-static", "Build static libraries (not recommended)"
+  option "with-docs", "Build documentation"
 
   depends_on "pkg-config" => :build
-  depends_on "autoconf" => :build
-  depends_on "automake" => :build
-  depends_on "libtool" => :build
+  depends_on "doxygen" => :build if build.with? "docs"
 
   def install
-    ENV.refurbish_args # needed for `./configure`
-
     args = %W[
       --prefix=#{prefix}
       --sysconfdir=#{etc}
       --localstatedir=#{var}
       --disable-dependency-tracking
+      --enable-android=no
+      --enable-audioeffects
+      --enable-clocksrc
+      --enable-components
+      --enable-debug=no
+      --enable-doc=#{build.with?("docs") ? "yes" : "no"}
+      --enable-static=#{build.with?("static") ? "yes" : "no"}
+      --enable-videoscheduler
```

Comment:
I'd suggest going with the upstream defaults, unless there's a particular motivation to deviate from the defaults.

Reply:
I suggest removing all the `--enable-*` options that are the default values. If they're not default, add a comment indicating why they're desirable.

Reply:
Um, I already did that. The remaining `--enable` options are there make builds reproducible.